[ffmpeg:x64-osx] build failure

**Host Environment**
- OS: macOS Big Sur 11.2.3
- Compiler: Apple clang version 12.0.0 (clang-1200.0.32.29)

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install ffmpeg[*]`

**Failure logs** 
-Cut and past the appropriate build messages from the console output.
```
./vcpkg install ffmpeg[*]
Computing installation plan...
The following packages will be built and installed:
    ffmpeg[all,all-gpl,all-nonfree,ass,avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,core,dav1d,fdk-aac,ffmpeg,ffplay,ffprobe,fontconfig,freetype,fribidi,gpl,iconv,ilbc,lzma,modplug,mp3lame,nonfree,nvcodec,opencl,opengl,openh264,openjpeg,openssl,opus,postproc,sdl2,snappy,soxr,speex,ssh,swresample,swscale,tensorflow,tesseract,theora,version3,vorbis,vpx,webp,x264,x265,xml2,zlib]:x64-osx -> 4.4#1
Detecting compiler hash for triplet x64-osx...
Could not locate cached archive: /Users/aaa/.cache/vcpkg/archives/c4/c4cc1e0e604510106130fe4ea4d09437d63505ec.zip
Starting package 1/1: ffmpeg:x64-osx
Building package ffmpeg[all,all-gpl,all-nonfree,ass,avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,core,dav1d,fdk-aac,ffmpeg,ffplay,ffprobe,fontconfig,freetype,fribidi,gpl,iconv,ilbc,lzma,modplug,mp3lame,nonfree,nvcodec,opencl,opengl,openh264,openjpeg,openssl,opus,postproc,sdl2,snappy,soxr,speex,ssh,swresample,swscale,tensorflow,tesseract,theora,version3,vorbis,vpx,webp,x264,x265,xml2,zlib]:x64-osx...
-- Using cached /Users/aaa/xxx/vcpkg/downloads/ffmpeg-ffmpeg-n4.4.tar.gz
-- Cleaning sources at /Users/aaa/xxx/vcpkg/buildtrees/ffmpeg/src/n4.4-b04690e0e5.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/aaa/xxx/vcpkg/downloads/ffmpeg-ffmpeg-n4.4.tar.gz
-- Applying patch 0001-create-lib-libraries.patch
-- Applying patch 0003-fix-windowsinclude.patch
-- Applying patch 0004-fix-debug-build.patch
-- Applying patch 0006-fix-StaticFeatures.patch
-- Applying patch 0007-fix-lib-naming.patch
-- Applying patch 0009-Fix-fdk-detection.patch
-- Applying patch 0010-Fix-x264-detection.patch
-- Applying patch 0011-Fix-x265-detection.patch
-- Applying patch 0012-Fix-ssl-110-detection.patch
-- Applying patch 0013-define-WINVER.patch
-- Applying patch 0014-avfilter-dependency-fix.patch
-- Applying patch 0015-Fix-xml2-detection.patch
-- Applying patch 0016-configure-dnn-needs-avformat.patch
-- Using source at /Users/aaa/xxx/vcpkg/buildtrees/ffmpeg/src/n4.4-b04690e0e5.clean
-- Building Options: --enable-pic --disable-doc --enable-debug --enable-runtime-cpudetect --enable-asm --enable-x86asm --enable-nonfree --enable-gpl --enable-version3 --enable-ffmpeg --enable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avformat --enable-avfilter --enable-postproc --enable-swresample --enable-swscale --enable-avresample --enable-libass --enable-avisynth --enable-bzlib --enable-libdav1d --enable-iconv --enable-libilbc --enable-libfdk-aac --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-lzma --enable-libmp3lame --enable-libmodplug --enable-cuda --enable-nvenc --enable-nvdec --enable-cuvid --enable-ffnvcodec --enable-opencl --enable-opengl --enable-libopenh264 --enable-libopenjpeg --enable-openssl --enable-libopus --enable-sdl2 --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtensorflow --enable-libtesseract --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-zlib --disable-vdpau  --pkg-config-flags=--static
-- Building Release Options:
-- Building ffmpeg for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:105 (message):
    Command failed: /bin/sh ./build.sh
    Working Directory: /Users/aaa/xxx/vcpkg/buildtrees/ffmpeg/x64-osx-rel
    Error code: 1
    See logs for more information:
      /Users/aaa/xxx/vcpkg/buildtrees/ffmpeg/build-x64-osx-rel-out.log

Call Stack (most recent call first):
  ports/ffmpeg/portfile.cmake:559 (vcpkg_execute_required_process)
  scripts/ports.cmake:139 (include)


Error: Building package ffmpeg:x64-osx failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: ffmpeg:x64-osx
  Vcpkg version: 2021-05-05-unknownhash

Additionally, attach any relevant sections from the log files above.
```
-Additional failure logs mentioned in the console output.
[build-x64-osx-rel-out.log](https://github.com/microsoft/vcpkg/files/6557700/build-x64-osx-rel-out.log)

**Additional context**
The libssh has been installed with` ./vcpkg install libssh` and `brew install libssh`. I also have tried `export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/Cellar/libssh/0.9.5_1/lib/pkgconfig ./vcpkg install ffmpeg[*]`, but still got the same error.

#### Experiment 1

1.1 First remove libssh

```
./vcpkg remove libssh
The following packages will be removed:
    libssh:x64-osx
Removing package libssh:x64-osx...
Removing package libssh:x64-osx... done
```
1.2 Then install ffmpeg

```
./vcpkg install ffmpeg[*]
Computing installation plan...
The following packages will be built and installed:
    ffmpeg[all,all-gpl,all-nonfree,ass,avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,core,dav1d,fdk-aac,ffmpeg,ffplay,ffprobe,fontconfig,freetype,fribidi,gpl,iconv,ilbc,lzma,modplug,mp3lame,nonfree,nvcodec,opencl,opengl,openh264,openjpeg,openssl,opus,postproc,sdl2,snappy,soxr,speex,ssh,swresample,swscale,tensorflow,tesseract,theora,version3,vorbis,vpx,webp,x264,x265,xml2,zlib]:x64-osx -> 4.4#1
  * libssh[core,crypto,mbedtls]:x64-osx -> 0.9.5#3
Additional packages (*) will be modified to complete this operation.
Detecting compiler hash for triplet x64-osx...
Could not locate cached archive: /Users/longshan/.cache/vcpkg/archives/60/60e07fb7884e614d14d5afd139ee61d433574127.zip
Could not locate cached archive: /Users/longshan/.cache/vcpkg/archives/c9/c9dbab220007912e7976648e8878cabed16a8f9b.zip
Starting package 1/2: libssh:x64-osx
Building package libssh[core,crypto,mbedtls]:x64-osx...
-- Using cached /Users/longshan/Work/Qt/Tiro/vcpkg/downloads/libssh-9c4af47965d284b2de26407bcd80473aba4ee4c9.tar.gz
-- Extracting source /Users/longshan/Work/Qt/Tiro/vcpkg/downloads/libssh-9c4af47965d284b2de26407bcd80473aba4ee4c9.tar.gz
-- Applying patch 0001-export-pkgconfig-file.patch
-- Applying patch 0002-mingw_for_Android.patch
-- Using source at /Users/longshan/Work/Qt/Tiro/vcpkg/buildtrees/libssh/src/3aba4ee4c9-5d3095f37d.clean
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Building x64-osx-dbg
-- Building x64-osx-rel
-- Fixing pkgconfig file: /Users/longshan/Work/Qt/Tiro/vcpkg/packages/libssh_x64-osx/lib/pkgconfig/libssh.pc
-- Fixing pkgconfig file: /Users/longshan/Work/Qt/Tiro/vcpkg/packages/libssh_x64-osx/debug/lib/pkgconfig/libssh.pc
-- Installing: /Users/longshan/Work/Qt/Tiro/vcpkg/packages/libssh_x64-osx/share/libssh/copyright
-- Installing: /Users/longshan/Work/Qt/Tiro/vcpkg/packages/libssh_x64-osx/share/libssh/usage
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: /Users/longshan/.cache/vcpkg/archives/60/60e07fb7884e614d14d5afd139ee61d433574127.zip
Building package libssh[core,crypto,mbedtls]:x64-osx... done
Installing package libssh[core,crypto,mbedtls]:x64-osx...
Installing package libssh[core,crypto,mbedtls]:x64-osx... done
Elapsed time for package libssh:x64-osx: 36.88 s
Starting package 2/2: ffmpeg:x64-osx
Building package ffmpeg[all,all-gpl,all-nonfree,ass,avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,core,dav1d,fdk-aac,ffmpeg,ffplay,ffprobe,fontconfig,freetype,fribidi,gpl,iconv,ilbc,lzma,modplug,mp3lame,nonfree,nvcodec,opencl,opengl,openh264,openjpeg,openssl,opus,postproc,sdl2,snappy,soxr,speex,ssh,swresample,swscale,tensorflow,tesseract,theora,version3,vorbis,vpx,webp,x264,x265,xml2,zlib]:x64-osx...
-- Using cached /Users/longshan/Work/Qt/Tiro/vcpkg/downloads/ffmpeg-ffmpeg-n4.4.tar.gz
-- Cleaning sources at /Users/longshan/Work/Qt/Tiro/vcpkg/buildtrees/ffmpeg/src/n4.4-b04690e0e5.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/longshan/Work/Qt/Tiro/vcpkg/downloads/ffmpeg-ffmpeg-n4.4.tar.gz
-- Applying patch 0001-create-lib-libraries.patch
-- Applying patch 0003-fix-windowsinclude.patch
-- Applying patch 0004-fix-debug-build.patch
-- Applying patch 0006-fix-StaticFeatures.patch
-- Applying patch 0007-fix-lib-naming.patch
-- Applying patch 0009-Fix-fdk-detection.patch
-- Applying patch 0010-Fix-x264-detection.patch
-- Applying patch 0011-Fix-x265-detection.patch
-- Applying patch 0012-Fix-ssl-110-detection.patch
-- Applying patch 0013-define-WINVER.patch
-- Applying patch 0014-avfilter-dependency-fix.patch
-- Applying patch 0015-Fix-xml2-detection.patch
-- Applying patch 0016-configure-dnn-needs-avformat.patch
-- Using source at /Users/longshan/Work/Qt/Tiro/vcpkg/buildtrees/ffmpeg/src/n4.4-b04690e0e5.clean
-- Building Options: --enable-pic --disable-doc --enable-debug --enable-runtime-cpudetect --enable-asm --enable-x86asm --enable-nonfree --enable-gpl --enable-version3 --enable-ffmpeg --enable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avformat --enable-avfilter --enable-postproc --enable-swresample --enable-swscale --enable-avresample --enable-libass --enable-avisynth --enable-bzlib --enable-libdav1d --enable-iconv --enable-libilbc --enable-libfdk-aac --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-lzma --enable-libmp3lame --enable-libmodplug --enable-cuda --enable-nvenc --enable-nvdec --enable-cuvid --enable-ffnvcodec --enable-opencl --enable-opengl --enable-libopenh264 --enable-libopenjpeg --enable-openssl --enable-libopus --enable-sdl2 --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtensorflow --enable-libtesseract --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-zlib --disable-vdpau  --pkg-config-flags=--static
-- Building Release Options:
-- Building ffmpeg for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:105 (message):
    Command failed: /bin/sh ./build.sh
    Working Directory: /Users/longshan/Work/Qt/Tiro/vcpkg/buildtrees/ffmpeg/x64-osx-rel
    Error code: 1
    See logs for more information:
      /Users/longshan/Work/Qt/Tiro/vcpkg/buildtrees/ffmpeg/build-x64-osx-rel-out.log

Call Stack (most recent call first):
  ports/ffmpeg/portfile.cmake:559 (vcpkg_execute_required_process)
  scripts/ports.cmake:139 (include)


Error: Building package ffmpeg:x64-osx failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: ffmpeg:x64-osx
  Vcpkg version: 2021-05-05-unknownhash

Additionally, attach any relevant sections from the log files above.
```

From the log, we can see the libssh is first installed.

#### Experiment 2
2.1 First remove libssh
```
./vcpkg remove libssh
The following packages will be removed:
    libssh:x64-osx
Removing package libssh:x64-osx...
Removing package libssh:x64-osx... done
```

2.2 Then install libssh
```
./vcpkg install libssh
Computing installation plan...
The following packages will be built and installed:
    libssh[core,crypto,mbedtls]:x64-osx -> 0.9.5#3
Detecting compiler hash for triplet x64-osx...
Using cached binary package: /Users/longshan/.cache/vcpkg/archives/60/60e07fb7884e614d14d5afd139ee61d433574127.zip
Starting package 1/1: libssh:x64-osx
Building package libssh[core,crypto,mbedtls]:x64-osx...
Building package libssh[core,crypto,mbedtls]:x64-osx... done
Installing package libssh[core,crypto,mbedtls]:x64-osx...
Installing package libssh[core,crypto,mbedtls]:x64-osx... done
Elapsed time for package libssh:x64-osx: 77.67 ms

Total elapsed time: 2.329 s

The package libssh is compatible with built-in CMake targets:

    find_package(libssh CONFIG REQUIRED)
    target_link_libraries(main PRIVATE ssh)

```

2.3 Finally install ffmpeg
```
./vcpkg install ffmpeg[*]
Computing installation plan...
The following packages will be built and installed:
    ffmpeg[all,all-gpl,all-nonfree,ass,avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,core,dav1d,fdk-aac,ffmpeg,ffplay,ffprobe,fontconfig,freetype,fribidi,gpl,iconv,ilbc,lzma,modplug,mp3lame,nonfree,nvcodec,opencl,opengl,openh264,openjpeg,openssl,opus,postproc,sdl2,snappy,soxr,speex,ssh,swresample,swscale,tensorflow,tesseract,theora,version3,vorbis,vpx,webp,x264,x265,xml2,zlib]:x64-osx -> 4.4#1
Detecting compiler hash for triplet x64-osx...
Could not locate cached archive: /Users/longshan/.cache/vcpkg/archives/c9/c9dbab220007912e7976648e8878cabed16a8f9b.zip
Starting package 1/1: ffmpeg:x64-osx
Building package ffmpeg[all,all-gpl,all-nonfree,ass,avcodec,avdevice,avfilter,avformat,avisynthplus,avresample,bzip2,core,dav1d,fdk-aac,ffmpeg,ffplay,ffprobe,fontconfig,freetype,fribidi,gpl,iconv,ilbc,lzma,modplug,mp3lame,nonfree,nvcodec,opencl,opengl,openh264,openjpeg,openssl,opus,postproc,sdl2,snappy,soxr,speex,ssh,swresample,swscale,tensorflow,tesseract,theora,version3,vorbis,vpx,webp,x264,x265,xml2,zlib]:x64-osx...
-- Using cached /Users/longshan/Work/Qt/Tiro/vcpkg/downloads/ffmpeg-ffmpeg-n4.4.tar.gz
-- Cleaning sources at /Users/longshan/Work/Qt/Tiro/vcpkg/buildtrees/ffmpeg/src/n4.4-b04690e0e5.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/longshan/Work/Qt/Tiro/vcpkg/downloads/ffmpeg-ffmpeg-n4.4.tar.gz
-- Applying patch 0001-create-lib-libraries.patch
-- Applying patch 0003-fix-windowsinclude.patch
-- Applying patch 0004-fix-debug-build.patch
-- Applying patch 0006-fix-StaticFeatures.patch
-- Applying patch 0007-fix-lib-naming.patch
-- Applying patch 0009-Fix-fdk-detection.patch
-- Applying patch 0010-Fix-x264-detection.patch
-- Applying patch 0011-Fix-x265-detection.patch
-- Applying patch 0012-Fix-ssl-110-detection.patch
-- Applying patch 0013-define-WINVER.patch
-- Applying patch 0014-avfilter-dependency-fix.patch
-- Applying patch 0015-Fix-xml2-detection.patch
-- Applying patch 0016-configure-dnn-needs-avformat.patch
-- Using source at /Users/longshan/Work/Qt/Tiro/vcpkg/buildtrees/ffmpeg/src/n4.4-b04690e0e5.clean
-- Building Options: --enable-pic --disable-doc --enable-debug --enable-runtime-cpudetect --enable-asm --enable-x86asm --enable-nonfree --enable-gpl --enable-version3 --enable-ffmpeg --enable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avformat --enable-avfilter --enable-postproc --enable-swresample --enable-swscale --enable-avresample --enable-libass --enable-avisynth --enable-bzlib --enable-libdav1d --enable-iconv --enable-libilbc --enable-libfdk-aac --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-lzma --enable-libmp3lame --enable-libmodplug --enable-cuda --enable-nvenc --enable-nvdec --enable-cuvid --enable-ffnvcodec --enable-opencl --enable-opengl --enable-libopenh264 --enable-libopenjpeg --enable-openssl --enable-libopus --enable-sdl2 --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtensorflow --enable-libtesseract --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-zlib --disable-vdpau  --pkg-config-flags=--static
-- Building Release Options:
-- Building ffmpeg for Release
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:105 (message):
    Command failed: /bin/sh ./build.sh
    Working Directory: /Users/longshan/Work/Qt/Tiro/vcpkg/buildtrees/ffmpeg/x64-osx-rel
    Error code: 1
    See logs for more information:
      /Users/longshan/Work/Qt/Tiro/vcpkg/buildtrees/ffmpeg/build-x64-osx-rel-out.log

Call Stack (most recent call first):
  ports/ffmpeg/portfile.cmake:559 (vcpkg_execute_required_process)
  scripts/ports.cmake:139 (include)


Error: Building package ffmpeg:x64-osx failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: ffmpeg:x64-osx
  Vcpkg version: 2021-05-05-unknownhash

Additionally, attach any relevant sections from the log files above.
```

They all have the same error:
```
=== CONFIGURING ===
ERROR: libssh not found using pkg-config

If you think configure made a mistake, make sure you are using the latest
version from Git.  If the latest version fails, report the problem to the
ffmpeg-user@ffmpeg.org mailing list or IRC #ffmpeg on irc.freenode.net.
Include the log file "ffbuild/config.log" produced by configure as this will help
solve the problem.
```

@ilongshan Can you confirm that you get similar errors in ./buildtrees/ffmpeg/x64-osx-rel/ffbuild/config.log (or even better, post the file)? In particular, it should be complaining about gssapi. Will post a patch to disable gssapi in libssh: the generated .pc file does not seem to support it.

```
require_pkg_config libssh libssh libssh/sftp.h sftp_init
check_pkg_config libssh libssh libssh/sftp.h sftp_init
test_pkg_config libssh libssh libssh/sftp.h sftp_init
pkg-config --exists --print-errors libssh
check_func_headers libssh/sftp.h sftp_init -I/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../include -L/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib -lssh -lmbedcrypto -lpthread
test_ld cc -I/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../include -L/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib -lssh -lmbedcrypto -lpthread
test_cc -I/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../include -L/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib
BEGIN /tmp/ffconf.YJTafjrc/test.c
    1   #include <libssh/sftp.h>
    2   #include <stdint.h>
    3   long check_sftp_init(void) { return (long) sftp_init; }
    4   int main(void) { int ret = 0;
    5    ret |= ((intptr_t)check_sftp_init) & 0xFFFF;
    6   return ret; }
END /tmp/ffconf.YJTafjrc/test.c
gcc -D_ISOC99_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_POSIX_C_SOURCE=200112 -D_XOPEN_SOURCE=600 -DPIC -std=c11 -fomit-frame-pointer -fPIC -pthread -I/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../include -L/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib -c -o /tmp/ffconf.YJTafjrc/test.o /tmp/ffconf.YJTafjrc/test.c
gcc -Wl,--as-needed -Wl,-z,noexecstack -I/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../include -L/home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib -o /tmp/ffconf.YJTafjrc/test /tmp/ffconf.YJTafjrc/test.o -lssh -lmbedcrypto -lpthread
/usr/bin/ld: /home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib/libssh.a(gssapi.c.o): in function `ssh_gssapi_log_error':
gssapi.c:(.text+0x5f): undefined reference to `gss_display_status'
/usr/bin/ld: gssapi.c:(.text+0x76): undefined reference to `gss_release_buffer'
/usr/bin/ld: gssapi.c:(.text+0x8b): undefined reference to `gss_release_buffer'
/usr/bin/ld: gssapi.c:(.text+0xc6): undefined reference to `gss_display_status'
/usr/bin/ld: /home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib/libssh.a(gssapi.c.o): in function `ssh_gssapi_free.isra.0':
gssapi.c:(.text+0x243): undefined reference to `gss_release_cred'
/usr/bin/ld: gssapi.c:(.text+0x25c): undefined reference to `gss_release_cred'
/usr/bin/ld: /home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib/libssh.a(gssapi.c.o): in function `ssh_gssapi_handle_userauth':
gssapi.c:(.text+0x406): undefined reference to `gss_create_empty_oid_set'
/usr/bin/ld: gssapi.c:(.text+0x41b): undefined reference to `gss_indicate_mechs'
/usr/bin/ld: gssapi.c:(.text+0x53e): undefined reference to `gss_test_oid_set_member'
/usr/bin/ld: gssapi.c:(.text+0x55a): undefined reference to `gss_add_oid_set_member'
/usr/bin/ld: gssapi.c:(.text+0x57e): undefined reference to `gss_release_oid_set'
/usr/bin/ld: gssapi.c:(.text+0x5d5): undefined reference to `GSS_C_NT_HOSTBASED_SERVICE'
/usr/bin/ld: gssapi.c:(.text+0x5dd): undefined reference to `gss_import_name'
/usr/bin/ld: gssapi.c:(.text+0x622): undefined reference to `gss_acquire_cred'
/usr/bin/ld: gssapi.c:(.text+0x635): undefined reference to `gss_release_name'
/usr/bin/ld: gssapi.c:(.text+0x642): undefined reference to `gss_release_oid_set'
/usr/bin/ld: gssapi.c:(.text+0x70c): undefined reference to `gss_test_oid_set_member'
/usr/bin/ld: gssapi.c:(.text+0x792): undefined reference to `gss_release_oid_set'
/usr/bin/ld: gssapi.c:(.text+0x8a5): undefined reference to `gss_release_oid_set'
/usr/bin/ld: gssapi.c:(.text+0x8d6): undefined reference to `gss_release_oid_set'
/usr/bin/ld: /home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib/libssh.a(gssapi.c.o): in function `ssh_packet_userauth_gssapi_token_server':
gssapi.c:(.text+0xbc3): undefined reference to `gss_accept_sec_context'
/usr/bin/ld: gssapi.c:(.text+0xc1a): undefined reference to `gss_display_name'
/usr/bin/ld: gssapi.c:(.text+0xc67): undefined reference to `gss_release_buffer'
/usr/bin/ld: /home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib/libssh.a(gssapi.c.o): in function `ssh_packet_userauth_gssapi_mic':
gssapi.c:(.text+0x1024): undefined reference to `gss_verify_mic'
/usr/bin/ld: /home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib/libssh.a(gssapi.c.o): in function `ssh_gssapi_auth_mic':
gssapi.c:(.text+0x1338): undefined reference to `GSS_C_NT_HOSTBASED_SERVICE'
/usr/bin/ld: gssapi.c:(.text+0x1349): undefined reference to `gss_import_name'
/usr/bin/ld: gssapi.c:(.text+0x13dd): undefined reference to `gss_inquire_cred'
/usr/bin/ld: gssapi.c:(.text+0x13fd): undefined reference to `gss_create_empty_oid_set'
/usr/bin/ld: gssapi.c:(.text+0x1461): undefined reference to `gss_inquire_cred_by_mech'
/usr/bin/ld: gssapi.c:(.text+0x147d): undefined reference to `gss_add_oid_set_member'
/usr/bin/ld: gssapi.c:(.text+0x150f): undefined reference to `GSS_C_NT_USER_NAME'
/usr/bin/ld: gssapi.c:(.text+0x1517): undefined reference to `gss_import_name'
/usr/bin/ld: gssapi.c:(.text+0x154b): undefined reference to `gss_acquire_cred'
/usr/bin/ld: gssapi.c:(.text+0x156b): undefined reference to `gss_release_name'
/usr/bin/ld: gssapi.c:(.text+0x1581): undefined reference to `gss_release_name'
/usr/bin/ld: /home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib/libssh.a(gssapi.c.o): in function `ssh_packet_userauth_gssapi_response':
gssapi.c:(.text+0x1b2a): undefined reference to `gss_init_sec_context'
/usr/bin/ld: /home/xxx/vcpkg/installed/x64-linux/lib/pkgconfig/../../lib/libssh.a(gssapi.c.o): in function `ssh_packet_userauth_gssapi_token_client':
gssapi.c:(.text+0x1db4): undefined reference to `gss_init_sec_context'
/usr/bin/ld: gssapi.c:(.text+0x1e8d): undefined reference to `gss_get_mic'
collect2: error: ld returned 1 exit status
ERROR: libssh not found using pkg-config
```

--- a/ports/libssh/CONTROL
+++ b/ports/libssh/CONTROL
@@ -1,6 +1,6 @@
 Source: libssh
 Version: 0.9.5
-Port-Version: 3
+Port-Version: 4
 Homepage: https://www.libssh.org/
 Build-Depends: libssh[core,mbedtls] (android)
 Description: libssh is a multiplatform C library implementing the SSHv2 protocol on client and server side

--- a/ports/libssh/portfile.cmake
+++ b/ports/libssh/portfile.cmake
@@ -32,7 +32,8 @@ vcpkg_configure_cmake(
         -DUNIT_TESTING=OFF
         -DCLIENT_TESTING=OFF
         -DSERVER_TESTING=OFF
-        -DWITH_NACL=OFF)
+        -DWITH_NACL=OFF
+        -DWITH_GSSAPI=OFF)
 
 vcpkg_install_cmake()
 vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT})

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3546,7 +3546,7 @@
     },
     "libssh": {
       "baseline": "0.9.5",
-      "port-version": 3
+      "port-version": 4
     },
     "libssh2": {
       "baseline": "1.9.0",

--- a/versions/l-/libssh.json
+++ b/versions/l-/libssh.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "0c5ced4b1fab4db90e2b032a3339a404cdc2f3ab",
+      "version-string": "0.9.5",
+      "port-version": 4
+    },
+    {
       "git-tree": "7436981a351a747649d03c4a27a06dc0bc3a0d16",
       "version-string": "0.9.5",
       "port-version": 3

